app/models/deploy_baseline.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class DeployBaseline
```

Comment:
this is some made up math, so explain what is going on in detail and why these metrics were chosen "it's the best we can do / it's what we want because"

Reply:
class name is kinda awkward ... DeployMetrics ?
... and call it `cycle time` ... and calculate it from first commit to prod deploy https://about.gitlab.com/2019/03/04/reduce-cycle-time/

Reply:
got it. thanks